tests/extras/extensions/test_ipython.py

```diff
@@ -218,3 +224,12 @@ def test_load_extension_not_in_kedro_project(self, mocker, caplog):
             "Make sure you run '%reload_kedro <project_root>'."
         )
         assert expected_message in log_messages
+
+    def test_load_extension_register_line_magic(self, mocker):
+        from IPython.testing.globalipapp import get_ipython
+
+        mocker.patch("kedro.extras.extensions.ipython.reload_kedro")
+
+        ipython = get_ipython()
+        load_ipython_extension(ipython)
+        ipython.magic("reload_kedro")
```

Comment:
Is it possible to test passing the arguments as well or would that require more of an end to end test setup?